scripts/elf_helper.py

```diff
@@ -210,15 +211,36 @@ def get_kobjects(self, addr):
 
 # --- helper functions for getting data from DIEs ---
 
+def die_get_spec(die):
+    if 'DW_AT_specification' not in die.attributes:
+        return None
+
+    spec_val = die.attributes["DW_AT_specification"].value
+
+    # offset of the DW_TAG_variable for the extern declaration
+    offset = spec_val + die.cu.cu_offset
+
+    if offset in extern_env:
```

Comment:
Minor, but you can just do `return extern_env.get(offset)` instead of this if/return combo.